### ISSUE TITLE
Accessing private raw photos is possible without authentication

### ISSUE THREAD

**Issue by qwerty287** [LycheeOrg/Lychee#1294]
### Detailed description of the problem [REQUIRED]
*A clear and concise description of what the bug is.*

### Steps to reproduce the issue
**Steps to reproduce the behavior:**
1. Log out of Lychee
2. open a raw thumbnail/photo (usually `/upload` paths)

If the photo isn't public, this shouldn't be accessible, even if the IDs and paths are not really predictable.

### Browser and system
Firefox 99.0.1 on Manjaro, server running with `php artisan serve`

**Comment by nagmat84** [comment]
Thanks for the bug report. We are aware of that  fact as it was a deliberate design decision and a trade-off between higher security and efficiency. If we wanted to authenticate every HTTP request for a media file, we had to route this request through the Laravel application and serve the file from PHP. For efficiency reasons these requests are directly handled by the web server and served from disk. Hence, they are not subject to the Laravel authentication mechanism.

As the photo IDs are random with 144bit entropy (at least for Lychee >= 4.5.0), the IDs provide a high level of unpredictability and we deemed this caveat acceptable.
